### PR TITLE
Removed null description @var in ApiResource

### PR DESCRIPTION
**Pull request by wizbit**
Doctrine's DocParser throws an exception if you set a description.

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #1050
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: tests/Annotation/ApiResourceTest.php [hunk 1]
```diff
@@ -14,9 +14,11 @@
 namespace ApiPlatform\Core\Tests\Annotation;
 
 use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\Common\Annotations\AnnotationReader;
 
 /**
  * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Marcus Speight <marcus@pmconnect.co.uk>
```
Comment (soyuka) [inline tests/Annotation/ApiResourceTest.php]: We're adding `@author` only when creating the file

File: tests/Annotation/AnnotatedClass.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ *     itemOperations={"foo":{"bar"}},
+ *     collectionOperations={"bar":{"foo"}},
+ *     attributes={"foo":"bar"}
+ * )
+ *
+ * @author Marcus Speight <marcus@pmconnect.co.uk>
+ */
+class AnnotatedClass
+{
+
```
Comment (soyuka) [inline tests/Annotation/AnnotatedClass.php]: no empty line, will fix travis (or run `php-cs-fixer`)

File: tests/Annotation/AnnotatedClass.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Annotation;
```
Comment (dunglas) [inline tests/Annotation/AnnotatedClass.php]: Missing blank line after

**Comment by dunglas** [comment]
Good catch! Can you run PHP CS Fixer and fix @soyuka's comments? I'll merge it in the release of today.

**Comment by wizbit** [comment]
@dunglas Ran PHP CS Fixer and fixed @soyuka's comments.

**Comment by dunglas** [comment]
Thank you @wizbit, nice fix!

**Pull request merged**